code/tier_classification/tier_classification.R

```diff
@@ -62,18 +73,12 @@ if(nrow(oncokb_anno) == 0){
     dplyr::filter(gnomAD_AF<0.01| is.na(gnomAD_AF)) %>%
     # filter on VAF over 5% to avoid background noise
     dplyr::filter(VAF>0.05) %>%
-    # filter to contain only mutations that has COSMIC ID
-    dplyr::mutate(has_COSMIC = dplyr::case_when(
-      stringr::str_detect(Existing_variation,
-                          "COSM") ~ "Yes",
-      TRUE ~ "No"
-    )) %>%
-    dplyr::filter(has_COSMIC == "Yes") %>%
+    # filter to contain only mutations that has COSMIC ID or has resistance mutation
+    dplyr::filter(grepl("COSM", Existing_variation) | cosmic_resistance_variant == "Yes") %>% 
```

Comment:
One thing I overlooked: can you place this COSMIC resistance variant criterion further up the pipeline? So rather than the logic being to look for 'COSM' or for the variant in the resistance file, the criterion would instead be: 

Tier 1 variants: a. Levels of evidence to include among Tier 1 would be: in the HIGHEST_LEVEL field, those with levels 1-3; OR in the Highest DX Level, levels 1-2; OR in the Highest PX Level, levels 1-2; OR a _Tier 1 cosmic resistance variant_.

So essentially editing line 70 with this?